resources/js/components/settings/requisition.tsx

```
@@ -0,0 +1,215 @@
+                        <span className="font-medium text-gray-900 dark:text-white">{requisition.user_language}</span>
+                    </div>
+                    <div className="flex justify-between">
+                        <span className="text-gray-500 dark:text-gray-400">Accounts:</span>
+                        <span className="font-medium text-gray-900 dark:text-white">{requisition.accounts?.length || 0}</span>
+                    </div>
+                </div>
+
+                {requisition.accounts && requisition.accounts.length > 0 && (
+                    <SimpleCollapse title={'Linked Accounts'} className="mt-3">
+                        {requisition.accounts.map((account_id: string) => (
+                            <AccountComponent account_id={account_id} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)